pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -43,10 +43,10 @@ type cadvisorClient struct {
 
 var _ Interface = new(cadvisorClient)
 
-// TODO(vmarmol): Make configurable.
-// The amount of time for which to keep stats in memory.
-const statsCacheDuration = 2 * time.Minute
-const maxHousekeepingInterval = 15 * time.Second
+// TODO: This code will likely be removed once cadvisor is a daemonset pod.
+var statsCacheDuration = flag.Duration("cadvisor_storage_duration", 2*time.Minute, "How long to keep data stored (Default: 2min).")
```

Comment:
I shudder at forcing the plumbing all the way through, when we are planning to rip out cadvisor.